README.md

```diff
@@ -1 +1,173 @@
-Start of the zendesk README
+# ZenDesk App
+
+## Quick Start
+
+- clone this repo
+- cp `.env.save` `.env`
+  - add your credentials to the `.env` file
+    - ZENDESK_URL - URL to zendesk account
+    - ZENDESK_USERNAME - your zendesk login username
+    - ZENDESK_API_TOKEN - generated via the following steps
+      - zendesk > Admin > CHANNELS > API > Settings
+      - Token Access > Enable
+      - Add API Token
+
+- `npm start` - start the node server
+- `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
+- `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+
+**Current Branches Needed**
+
+The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+
+mm-webapp: `apps-modals`  
+mm-plugin-apps: `apps-modals`
+
+**Install the app**
+
+`/apps install --url http://localhost:4000/manifest.json --app-secret thisisthesecret`
+
+  After installing the app, a provisioned bot account will be created for user
+  @zendesk and posted in a DM.
+
+  The following values are stored locally in `app_store.json`
+
+- `bot_access_token`
+- `oauth2_client_secret`
+
+## Create a ticket
+
+EndPoint: `http://localhost:4000/createform`
+Method: `post`
+Body (example):
+
+```json
+{
+    "channel_id": "rgiqcxrm8jdjzgj536gb45oh3e",
+    "user_id": "6fiyj9ni9t835dnbni1ddrj93y",
+    "submission": {
```

Comment:
Should this be updated to use the new Apps form structure?

Reply:
I would say no and will be deleting this altogether.  I will replace with an example of the create ticket modal instructions.  This info is left over from when I was first creating the app and using the README as a place to store random information as I was discovering/learning the API.